README.md

```diff
@@ -25,235 +24,143 @@ LFortran, see that project for a list of sponsors.
+            ```bash
+            conda create -n lp -c conda-forge llvmdev=11.0.1 bison=3.4 re2c python cmake make toml clangdev git
+            ```
+        
+        - Optionally, you can change the directory to a Windows location using `cd /mnt/[drive letter]/[windows location]`. For e.g. - `cd mnt/c/Users/name/source/repos/`.
 
+    
+- ### Clone the LPython repository
+    Make sure you have `git` installed. Type the following command to clone the repository:
 
-  ./build0.sh
-  ./build1.sh
-  ```
-
-### Windows
-
-- Create a Conda environment using the pre-existing file:
-
-  ```bash
-  conda env create -f environment_win.yml
-  conda activate lp
-  ```
-
-- Generate prerequisite files; build in Release Mode:
-
-  ```bash
-  call build0.bat
-  call build1.bat
-  ```
-
-- Tests and examples
-
-  ```bash
-  ctest
-  src\bin\lpython examples\expr2.py
-  src\bin\lpython examples\expr2.py -o a.out
-  a.out
-  ```
-
-- Whenever you are updating a test case file, you also need to update all the reference results associated with that test case:
-
-  ```
-  python run_tests.py -u --skip-run-with-dbg
-  ```
-
-- To see all the options associated with LPython test suite, use:
-
-  ```
-  python run_tests.py --help
-  ```
-
-## Tests:
-
-### Linux or MacOS
-
-- Run tests:
-
-  ```bash
-  ctest
-  ./run_tests.py
-  ```
-
-- Update test references:
-  ```
-  ./run_tests.py -u
-  ```
-
-- Run integration tests:
-
-  ```bash
-  cd integration_tests
-  ./run_tests.py
-  ```
-- Troubleshooting on MacOS latest version:
-  - In case of recently updated MacOS, you may get a warning like below in some test cases:
     ```bash
-    ld: warning: object file (test_list_index2.out.tmp.o) was built for newer macOS version (14.0) than being linked (13.3)
-    ```
-    This leads to mismatch of hashes with expected output in some test cases, this can be resolved by updating command line tools. Below is a snippet for the same.
-
-    ```bash
-    git clean -dfx
-    sudo rm -rf /Library/Developer/CommandLineTools # make sure you know what you're doing here
-    sudo xcode-select --install
-    ./build.sh
-    ./run_tests.py
+    git clone https://github.com/lcompilers/lpython.git
+    cd lpython
     ```
+    
+    You may also use GitHub Desktop to do the same.
 
+## Building LPython
+- ### Linux and macOS
+    - Create a Conda environment:
 
-### Windows
+        ```bash
+        conda env create -f environment_unix.yml
+        conda activate lp
+        ```
 
-- Run integration tests
+    - Generate the prerequisite files and build in Debug Mode:
 
-  ```bash
-  python run_tests.py --skip-run-with-dbg
-  ```
+        ```bash
+        # if you are developing on top of a forked repository; please run following command first
+        # ./generate_default_tag.sh
 
-- Update reference tests
 
-  ```bash
-  python run_tests.py -u --skip-run-with-dbg
-  ```
+        ./build0.sh
+        ./build1.sh
+        ```
 
-## Speed up Integration Tests on MacOS
```

Comment:
@kmr-srbh some of the information seems to be removed from the previous README.md. 
Can you please look into it and add them back?

`Speed up Integration Tests on MacOS`
`Examples`
`Troubleshooting on MacOS latest version`
`Tests`

Reply:
@Thirumalai-Shaktivel that section was moved to the installation docs.

Reply:
Okay, then it's fine.
Yes, I can see it here: https://github.com/lcompilers/lpython/blob/main/doc/src/installation.md

Reply:
Updated here: https://github.com/lcompilers/lpython/pull/2537